js/views/media_overlay_data_injector.js

```diff
@@ -1,422 +1,445 @@
 //  Copyright (c) 2014 Readium Foundation and/or its licensees. All rights reserved.
-//  
```

Comment:
Code auto-format should not be committed in this PR (if there is a significant and PR-relevant code change, we need to narrow it down to a palatable number of lines).